lib/Driver/ToolChains.cpp

```diff
@@ -1051,6 +1062,18 @@ static void getRuntimeStaticLibraryPath(SmallVectorImpl<char> &runtimeLibPath,
                           getPlatformNameForTriple(TC.getTriple()));
 }
 
+/// On Darwin we link fat libs in the root of the library path and on other OS
+/// targets we look up libraries in the target arch's subfolder.
+static void getRuntimeStaticLibraryPathWithArch(
+                                  SmallVectorImpl<char> &runtimeLibPath,
```

Comment:
Nitpick: rather than right-aligning, we just double-indent these.

Reply:
fixed

Reply:
fixed.